### PR TITLE
[FEATURE] Link to domain verification on dev portal

### PR DESCRIPTION
**Pull request by stevenzeiler**
We will have Eli and Travis go over the user experience here, but at least it gets links to the build portal in place of unverified domains.

### CODE REVIEW

**Comment by stevenzeiler** [comment]
:+1:

File: views/validator.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 
 <div class='row'>
   <h1>{{validatorPublicKey}}</h1>
-  <h3 class='validator-show-domain'>{{domain}}</h3>
+  <h3 ng-if="domain" class='validator-show-domain'>{{domain}}</h3>
+  <h3 ng-if="!domain" class='validator-show-domain'>Is this your validator? <a style='text-decoration:underline; font-weight:bold' href='https://ripple.com/build/rippled-apis/rippled-setup/#domain-verification' target='_blank'>Verify Domain</a></h3>
```
Comment (wilsonianb) [inline views/validator.html:5]: Maybe `Verify Domain` -> `Verify Your Domain`

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**